### DISCUSSION TITLE
DEPRECATED. Path to languagetool-server.jar on your local machine

### DISCUSSION THREAD

**Discussion by aisbergde** [davidlday/vscode-languagetool-linter#287] [Q&A] [answered]
I followed the instruction:
Option 2: Use an Extension-Managed Service

But in the settings, it is marked as deprecated.

Why it is deprecated and what should I do now to use the extension?
Will the "option 2" be removed in the future?

**Comment by davidlday** [comment]
Great question. I need to update the documentation. The only piece deprecated for now is using the `Language Tool Linter > Managed: Jar File` setting. Please use the `Language Tool Linter > Managed: Class Path` setting just above it instead.

Hopefully that answers your question. If not, let me know.

**Comment by aisbergde** [comment]
Currently, I am using the jar file. It (still) works, but I get warnings, that I should not use it.

The question is: How to use the "Class Path"? What is a class, wich path should be set?

I used chocolatey to install, the installation is located here: C:\ProgramData\chocolatey\lib\languagetool\tools\LanguageTool-5.1

**Comment by davidlday** [comment, marked as answer]
Class Path allows for multiple java archives (jar files) to be passed to Java, where the original only allowed for one jar file. In your case, you can just copy the path to the jar file from the old setting to the "Class Path" setting and it will work the same. I added this as some package managers don't install LanguageTool as a single jar file, but as multiple jar files that all need included in the class path.
